graylog2-web-interface/src/components/common/Spinner.test.jsx

```diff
@@ -1,17 +1,21 @@
 import React from 'react';
-import { mount } from 'wrappedEnzyme';
+import { render, cleanup } from 'wrappedTestingLibrary';
```

Comment:
Should we have a test for the `delay`?

Reply:
I added a test, it only tests styling, but I agree it makes sense.